schema/artist/index.js

```diff
@@ -238,6 +238,14 @@ const ArtistType = new GraphQLObjectType({
           gravity(`artist/${id}/artworks`, options)
             .then(exclude(options.exclude, 'id')),
       },
+      formatted_artworks_count_string: {
```

Comment:
Is the `_string` suffix used elsewhere? I’m generally not a fan of Hungarian notation, in this case especially as the schema specifies it is a string.